addon/components/print-course.hbs

```diff
@@ -345,117 +194,289 @@
       </div>
     </section>
-    <section class="block">
-      <div class="title">
-        {{t "general.terms"}} ({{session.terms.length}})
-      </div>
-      <div class="content">
-        <div class="content">
-          {{#each (await session.associatedVocabularies) as |vocab|}}
-            <DetailTermsList
-              @vocabulary={{vocab}}
-              @terms={{session.terms}}
-              @canEdit={{false}}
-            />
-          {{/each}}
-        </div>
-      </div>
-    </section>
-    <section class="block">
+    <section class="block" data-test-course-mesh>
       <div class="title">
-        {{t "general.mesh"}} ({{session.meshDescriptors.length}})
+        {{t "general.mesh"}} ({{@course.meshDescriptors.length}})
       </div>
       <div class="content">
         <ul class="inline-list">
-          {{#each (sort-by "title" (await session.meshDescriptors)) as |descriptor|}}
+          {{#each (sort-by "title" (await @course.meshDescriptors)) as |descriptor|}}
             <li>
               {{descriptor.name}}
             </li>
           {{/each}}
         </ul>
       </div>
     </section>
-    {{#if session.isIndependentLearning}}
-      <section class="block" data-test-session-ilm-section>
+    {{#each (sort-by "title" this.sessions) as |session|}}
+      <div class="header" data-test-session-header>
+        <h2>
+          {{session.title}}
+        </h2>
+        <PublicationStatus @item={{session}} @showIcon={{true}} @showText={{true}} />
+      </div>
+      <section class="overview block">
         <div class="title">
-          {{t "general.independentLearning"}}
+          {{t "general.overview"}}
         </div>
         <div class="content">
-          <table>
-            <thead>
-              <tr>
-                <th>
-                  {{t "general.hours"}}
-                </th>
-                <th>
-                  {{t "general.dueBy"}}
-                </th>
-              </tr>
-            </thead>
-            <tbody>
-              <tr>
-                <td>
-                  {{get (await session.ilmSession) "hours"}}
-                </td>
-                <td>
-                  {{format-date (get (await session.ilmSession) "dueDate")}}
-                </td>
-              </tr>
-            </tbody>
-          </table>
+          <div class="inline-label-data-block">
+            <label>
+              {{t "general.sessionType"}}:
+            </label>
+            <div>
+              {{session.sessionType.title}}
+            </div>
+          </div>
+          <br>
+          <div class="inline-label-data-block">
+            <label for="supplemental-curriculum-{{templateId}}">
+              {{t "general.supplementalCurriculum"}}:
+            </label>
+            <div>
+              <input
+                id="supplemental-curriculum-{{templateId}}"
+                type="checkbox"
+                checked={{session.supplemental}}
+                disabled="disabled"
```

Comment:
Things I learned `disabled="disabled"` is the most semantically valid way to specify this 🤔

Reply:
:+1: